.github/workflows/load-tests.yml

```diff
@@ -78,25 +78,12 @@ jobs:
         working-directory: tools/ci-k6
         run: npm ci
 
-      - name: Set up Docker Buildx
-        if: ${{ steps.should.outputs.RUN == 'true' }}
-        uses: docker/setup-buildx-action@v3
-        # Use GitHub Container Registry instead due to rate limits
-        with:
-          buildkitd-config-inline: |
-            [registry."ghcr.io"]
-      - name: Login to DockerHub
```

Comment:
We still need to setup to use ghcr don't we? Otherwise we'll hit the rate limits of docker hub again right?

Reply:
Ah, okay. But do we need to login? We're not pushing anything.

Reply:
No. As long as we are pulling from ghcr, we don't need to.